src/components/SkeletonLoader/SkeletonLoader.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsxImportSource theme-ui */
+import React from "react"
+import ContentLoader from "react-content-loader"
+
+export interface SkeletonLoaderProps {
+  variant?: string
+  width?: string
+  height?: string
+  radius?: string
+  top?: string
+  left?: string
```

Comment:
S: I think all of these make more sense as type `number`, no? Only maybe radius could be `string | number`